### ISSUE TITLE
DownloadBuildArtifactsV0 stopped working at v.0.194.0 on Windows Server 2008 R2 private agent

### ISSUE THREAD

**Issue by nicolasbouchard1412** [microsoft/azure-pipelines-tasks#15459]
**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: DownloadBuildArtifactsV0 V0.194.0

## Environment
- Azure Pipelines
- Organization: familiprix
- Team project: Équipe web
- Build: 20211019.6
- Release: Release-151

- Agent - Private: Private
    - Agent version: 2.194.0 (same problem with 2.193.0)
    - OS: Windows Server 2008 R2

## Issue Description

The task DownloadBuildArtifactsV0 stopped working on our private agent since the agent have updated the said task to version 0.194.0. I'm sure it's the new version that caused the problem because I had exactly the same problem on another server after the Task update, plus the pipeline hasn't been edited in a long time.

So basically, the Task run successfully but download nothing. On the server, the folder "a" is indeed empty.
![image](https://user-images.githubusercontent.com/43881677/139327417-dcda3305-273d-4d52-8ab2-4fc215f0dd3e.png)

Here is the exact same download Task in a previous release where v0.193.0 was used.
![image](https://user-images.githubusercontent.com/43881677/139327461-2c3a34bd-bf33-477e-9a9f-cb9897e2689e.png)

Note that I've verified others pipelines and some are working fine with the v0.194.0, which let me think is the OS that is now incompatible with the Task. Problem is we can't specify a Task version in the Classic Release Pipeline, so we're stuck.

Thanks

**Comment by EzzhevNikita** [comment]
Hey @nicolasbouchard1412,  Is the problem still actual for you? We aren't able to reproduce it currently. You have mentioned that the issue happens only for some agents. Could you please share information about agents' OSes, where the task is working fine, and where it is failing? Also, it will be cool to see debug logs for the failed and successful runs.

**Comment by EzzhevNikita** [comment]
Hi @nicolasbouchard1412, do you still need some help with this issue?

**Comment by nicolasbouchard1412** [comment]
Hi @EzzhevNikita,

I have managed to resolve the issue by changing the [Publish Build Artifacts task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-build-artifacts?view=azure-devops) for the [Publish Pipeline Artifacts](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/pipeline-artifacts?view=azure-devops). 

Since the Download Pipeline Artifacts Task is not available on classic release pipeline, I also had to migrate to a multi-stage YAML pipeline.

Thank you, you can close the issue.
